Automattic/newspack-popups
# Single GET per page; lightweight POST endpoint

### All Submissions:






### Changes proposed in this Pull Request:



Switches the plugin to use lightweight API, adds POST requests handling; changes how `amp-access` is used – to make just one GET request per page.

### How to test the changes in this Pull Request:

1. Create multiple campaigns
2. On a page where they all appear, observe that one GET request is made to get campaigns visibility
3. Dismiss a campaign – observe the POST request is made to the lighweight API endpoint
4. Observe all campaign functionality related to display conditions is working as before

### Other information:
